tideways-proxy/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+extra="${PROXY_OPTIONS} --hostname=${HOSTNAME:-tideways}"
+extra="${extra} --listen=0.0.0.0:${TIDEWAYS_PORT}"
```

Comment:
The beautiful spelling would be`extra+=" --listen=0.0.0.0:${TIDEWAYS_PORT}"`

Reply:
Ah, didn't know that also works - will change it 👍